Small little amendments to the topics within the conceptual section … (#7)
postsharp/Metalama.Documentation: conceptual/deployment/deployment.md

# Deployment and Configuration

> <!--- I wonder if the following might be better:

Reviewer: ```suggestion
```

> This section describes the component parts that make up Metalama, how to install them, and how to configure Metalama before use. -->

Reviewer: ```suggestion
```

> This section describes the components that make Metalama, how to install them, and how to configure Metalama before use.

Reviewer: ```suggestion
This section describes the component parts that make up Metalama, how to install them, and how to configure Metalama before use.
```
